examples/requests/plane/main.js

```diff
@@ -0,0 +1,19 @@
+Core.processGlobal();
+Plane.askPermission();
+
+if (!Plane.started) {
```

Comment:
This code can be simplified. I'd rather moved waiting for a permission inside Plain askPermission function.